apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs

```diff
@@ -276,5 +278,190 @@ private static string ValidateTableName(string table, string parameterName)
             }
             return table;
         }
+
+        private static readonly char s_singleQuote = '\'';
+        private static readonly char s_doubleQuote = '"';
+        private static readonly char s_backtickQuote = '`';
+        private static readonly char s_hyphen = '-';
+        private static readonly char s_dash = '#';
```

Comment:
I think this is sharp?
```suggestion
        private static readonly char s_sharp = '#';
```

Reply:
TBH, I'm open to calling it anything =) But formally this is the number sign, _hash_ (so certainly not dash...) or pound sign, while the sharp sign is slightly different: https://en.wikipedia.org/wiki/Number_sign and https://en.wikipedia.org/wiki/Sharp_(music)